doc/users/interactive.rst

```diff
@@ -57,64 +58,58 @@ IPython integration
 ===================
 
 We recommend using IPython for an interactive shell.  In addition to
-all of its features (improved tab-completion, magics,
-multiline editing, etc), it also ensures that the GUI toolkit event
-loop is properly integrated with the command line (see
-:ref:`cp_integration`).  To configure the integration and enable
-:ref:`interactive mode <controlling-interactive>` use the
-``%matplotlib`` magic
+all of its features (improved tab-completion, magics, multiline editing, etc), 
+it also ensures that the GUI toolkit event loop is properly integrated 
+with the command line (see :ref:`cp_integration`). 
+
+In this example, we create and modify a figure via an IPython prompt. 
+The figure displays in a Qt5Agg GUI window. To configure the integration 
+and enable :ref:`interactive mode <controlling-interactive>` use the
+``%matplotlib`` magic:
 
 .. highlight:: ipython
 
 ::
-
-   user@machine:~ $ ipython
-   Python 3.8.2 (default, Apr  8 2020, 14:31:25)
-   Type 'copyright', 'credits' or 'license' for more information
-   IPython 7.13.0 -- An enhanced Interactive Python. Type '?' for help.
-
    In [1]: %matplotlib
    Using matplotlib backend: Qt5Agg
 
    In [2]: import matplotlib.pyplot as plt
 
-Calling
+Create a new figure window:
 
 ::
 
    In [3]: fig, ax = plt.subplots()
 
-will pop open a window for you and
+
+Add a line plot of the data to the window:
 
 ::
 
    In [4]: ln, = ax.plot(range(5))
 
-will show your data in the window.  If you change something about the
-line, for example the color
+Change the color of the line from blue to orange:
 
 ::
 
    In [5]: ln.set_color('orange')
 
-it will be reflected immediately. If you wish to disable this behavior
-use
+If you wish to disable interactive modification of the plot:
```

Comment:
It is the automatic re-draw that is disabled, if you have a prompt you can always interactively modify it (even with out the event loop integration), the user then has to intentionally trigger re-draws or spin the event loop.

Reply:
I'm using interactive modification == automatic redraw, but will clarify that